src/main/java/marquez/resources/JobRunResource.java

```diff
@@ -0,0 +1,147 @@
+package marquez.resources;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static marquez.api.JobRunState.State.fromInt;
+import static marquez.api.JobRunState.State.toInt;
+
+import com.codahale.metrics.annotation.Timed;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.sql.Timestamp;
+import java.util.UUID;
+import javax.validation.Valid;
+import javax.ws.rs.*;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Response;
+import marquez.api.*;
+import marquez.db.dao.JobRunDAO;
+
+@Path("/job_runs")
+@Produces(APPLICATION_JSON)
+public class JobRunResource extends BaseResource {
+
+  private final JobRunDAO dao;
+
+  public JobRunResource(final JobRunDAO dao) {
+    this.dao = dao;
+  }
+
+  @POST
+  @Consumes(APPLICATION_JSON)
+  @Timed
+  public Response create(@Valid CreateJobRunRequest request) {
+    UUID jobRunGuid = UUID.randomUUID();
+
+    // TODO: Validate that the job run definition exists. Otherwise, throw a 404.
+    // This will be much easier to implement once JobRunDefinition is checked in.
+    try {
+      JobRun jobRun =
+          new JobRun(
+              jobRunGuid,
+              new Timestamp(System.currentTimeMillis()),
+              null,
+              null,
+              request.getJobRunDefinitionGuid(),
+              request.getState());
```

Comment:
TODO(code review): hard-code this to NEW state (see complementary comment above).